### ISSUE TITLE
Youtube autoplay on desktop chrome

### ISSUE THREAD

**Issue by johnaschroeder** [henrygd/bigpicture#23]
First, thanks for a nice library, much appreciated.  It appears that youtube videos now require two clicks to play on a desktop browser.  I know mobile safari prohibits this, but it was working previously on the desktop with chrome:  click once, bigpicture launches and the video begins playing.   I'm using 
```
class="vid iframevid" 
iframesrc="https://www.youtube.com/embed/{id-here}?html5=1&rel=0&playsinline=1&autoplay=1&modestbranding=1"
``` 
The youtube example in your documentation also requires two clicks to play, once to launch the bigpicture modal and a second to start the youtube video.  Vimeo works with one click.  That makes me think there's an issue here.  As a reference, fancybox does autoplay, so it's not a change in chrome or youtube.  Example here: [https://fancyapps.com/fancybox/3/](url)

**Comment by henrygd** [comment]
Would you mind trying this and letting me know if it resolves the issue?

[https://bptest.surge.sh](https://bptest.surge.sh)

**Comment by johnaschroeder** [comment]
Yep, youtube is now a single click to play.  Thanks for jumping on that, can you share the solution?  The second video with youtube in an iframe looks like the same iframeSrc as i have. Code change?

**Comment by henrygd** [comment]
Code change. Just needed to add an extra allow attribute to the iframe as [outlined by google](https://developers.google.com/web/updates/2017/09/autoplay-policy-changes) a while back. Something I should have done earlier, but apparently wasn't enforced for youtube until recently.

I'll update and release w/ the fix today or tomorrow. Thanks for bringing this to attention.

```html
<!-- autoplay no longer allowed -->
<iframe src="https://cross-origin.com/myvideo" allowfullscreen>

<!-- autoplay allowed -->
<iframe src="https://cross-origin.com/myvideo" allowfullscreen allow="autoplay; fullscreen">
```

**Comment by johnaschroeder** [comment]
Much appreciated!

**Comment by henrygd** [comment]
Released [2.1.1](https://github.com/henrygd/bigpicture/releases/tag/v2.1.1) which should fix this issue. Let me know if you run into it again.
